### ISSUE TITLE
Document BaRatinAGE folder structure and bam file format

### ISSUE THREAD

**Issue by benRenard** [BaRatin-tools/BaRatinAGE#43]
If this folder is really useful, then we should explain why and how. But is it ?

**Comment by benRenard** [comment]
In fact I think it might have been used by quite a few advanced users of BaRatin, including Jerome, Felipe, Emeline... @JeromeLeCoz, what do you think?

In v3, there is no 'recycle' folder but it's easy to find all runs of the current BaRatinAGE session in exe/bam_workspace, which serves the same purpose and is perfectly fine in my eyes. But it's still worth mentioning this in the documentation and referring to https://github.com/BaM-tools/BaMdocs for an explanation of the file contents (and the guides therein will have to be translated from doc/pdf to markdown/html).

I'm leaving the ticket opened - but no code change needed, it's a purely 'documentation' thing.

**Comment by IvanHeriver** [comment]
Indeed, all results of the current session can be found in the folder `exe/bam_workspace/xxx/zzz` where `xxx` corresponds to the ID of the session (the easiest way to know which folder is the creation date) and `zzz` the BaM run ID (the format of the ID can be used to know when the run was done: `yyyyMMdd_hhmmss_xxxxx` where the letters `yMdhms` are for date and time.

However, session folder is deleted when the session terminates (when BaRatinAGE is closed, a session represent an instance of BaRatinAGE since multiple instances can co-exist).

Raw results of BaM can be also retrieved from the `.bam` project file (after unzipping it). Each run is stored in another nested zip file with the name format as described above. It is however not always so straigthforward to know which folder correspond to which run. The file main_config.json contains information about each BaRatinAGE component including which Bam run ID correspond to which component.

To conclude this comment, I'd say that documentation could also include a description of the `.bam` file format.

**Comment by IvanHeriver** [comment]
content of a BaM file is now documented here: https://baratin-tools.github.io/fr/doc/baratinage/aller-plus-loin/
